platform/android/export/export.cpp

```diff
@@ -1564,7 +1564,7 @@ class EditorExportAndroid : public EditorExportPlatform {
 				_fix_resources(p_preset, data);
 			}
 
-			if (file == "res/drawable/icon.png") {
+			if (file == "res/drawable-nodpi-v4/icon.png") {
```

Comment:
You renamed the actual folder to `drawable-nodpi`, which is correct?

Reply:
yeap. I need to change this to update icon properly.
it comes with fixing `res/drawable/icon.png` warning.

Reply:
yes. not specified dpi resource should be under `drawable-nodpi` now.

Reply:
But I mean, the file is in `res/drawable-nodpi/icon.png`, but you're referencing `drawable-nodpi-v4` here, isn't this `-v4` an issue?

Reply:
after making `.apk` for templates, it changes to `drawable-nodpi-v4`

Reply:
Ok, thanks for the explanation :)